xskillscore/core/probabilistic.py

```diff
@@ -1153,9 +1153,9 @@ def _auc(fpr, tpr, dim="probability_bin"):
                 area = xr.apply_ufunc(
                     np.trapz, tpr, fpr, input_core_dims=[[dim], [dim]], dask="allowed"
                 )
-    area = np.abs(area)
+    area = abs(area)
```

Comment:
isnt np faster?

Reply:
I see no difference in speed

Reply:
It calls `__abs__` (unlike using the builtin `sum`, which would be much slower)